.github/workflows/ci.yml

```diff
@@ -15,16 +15,20 @@ jobs:
         with:
           node-version: ${{ matrix.node-version }}
       - name: Restore cached dependencies
-        uses: actions/cache@v1
+        uses: actions/cache@v2
         with:
           path: node_modules
           key: node-modules-${{ hashFiles('package.json') }}
       - name: Install dependencies
         run: npm install
       - name: Run Tests
         run: npm run ci
-      # - name: Upload coverage to Codecov
-      #   uses: codecov/codecov-action@v1
-      #   with:
-      #     file: ./coverage/coverage-final.json
-      #     token: ${{ secrets.CODECOV_TOKEN }}
+
+  automerge:
+    needs: build
+    runs-on: ubuntu-latest
+    steps:
+      - uses: fastify/github-action-merge-dependabot@v1
+        if: ${{ github.actor == 'dependabot[bot]' && github.event_name == 'pull_request' }}
+        with:
+          github-token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
can you please also fix the repo you got the wrong version of this from? I believe there's a card for it https://github.com/orgs/nearform/projects/3#card-55207086

Reply:
Sure, I will update that as well 👍

Reply:
Fixed here: https://github.com/nearform/fastify-secrets-azure/pull/28